### PR TITLE
Makefile: create distribution rules for wheel packages

### PR DESCRIPTION
**Pull request by clebergnu**
Python wheel packages are really convenient.  Let's create and
distrubute them in addition to the source packages.

Signed-off-by: Cleber Rosa <crosa@redhat.com>

### CODE REVIEW

**Comment by clebergnu** [comment]
[Publish Avocado in "wheel" format in PyPI](https://trello.com/c/jUrIcqQY/1210-publish-avocado-in-wheel-format-in-pypi)

File: Makefile [hunk 1]
```diff
@@ -71,7 +71,19 @@ source-pypi: clean
...
+	for PLUGIN in $(AVOCADO_OPTIONAL_PLUGINS); do\
+		if test -f $$PLUGIN/setup.py; then\
+			echo ">> Creating wheel distribution for $$PLUGIN";\
+			cd $$PLUGIN;\
+			$(PYTHON) setup.py bdist_wheel -d ../../PYPI_UPLOAD;\
+			cd -;\
+                fi;\
+	done
+
+pypi: wheel source-pypi develop
```
Comment (apahim) [inline Makefile]: cool! I'm assuming, since you already uploaded the wheel packages, that's the verification we need.

File: Makefile [hunk 2]
```diff
@@ -71,7 +71,19 @@ source-pypi: clean
...
 
-pypi: source-pypi develop
+wheel: clean
+	if test ! -d PYPI_UPLOAD; then mkdir PYPI_UPLOAD; fi
+	$(PYTHON) setup.py bdist_wheel -d PYPI_UPLOAD
+	for PLUGIN in $(AVOCADO_OPTIONAL_PLUGINS); do\
+		if test -f $$PLUGIN/setup.py; then\
+			echo ">> Creating wheel distribution for $$PLUGIN";\
+			cd $$PLUGIN;\
+			$(PYTHON) setup.py bdist_wheel -d ../../PYPI_UPLOAD;\
```
Comment (apahim) [inline Makefile]: `pip install wheel` is a requirement here. Are we going to manage that requirement?

Comment (clebergnu) [reply Makefile]: That is a development requirement... not sure we should be tracking that.  It's akin to requiring `git` or `rpmbuild`.

**Pull request merged**